examples/speaker_recognition/conf/speaker_diarization.yaml

```diff
@@ -15,11 +15,17 @@ diarizer:
     model_path: null #.nemo local model path or pretrained model name or none
     window_length_in_sec: 0.15
     shift_length_in_sec: 0.01
-    threshold: 0.7 # tune threshold on dev set. Check <NeMo_git_root>/scripts/voice_activity_detection/vad_tune_threshold.py
     vad_decision_smoothing: True
     smoothing_params:
       method: "median" 
       overlap: 0.875
+    threshold: null # if not null, use this value and ignore the following onset and offset. Warning! it will be removed in release 1.5!
+    postprocessing_params: # Tuned parameter for CH109! (18 sessions as dev set) You might need to choose resonable threshold or tune the threshold on your dev set. Check <NeMo_git_root>/scripts/voice_activity_detection/vad_tune_threshold.py
+      onset: 0
+      offset: 0.8
```

Comment:
aren't we merging onset and offset to reduce # of parameters and have just threshold?

Reply:
Discussed